FredLin.md

```diff
@@ -1,14 +1,22 @@
 [Open bugs assigned to me](https://bugzilla.mozilla.org/buglist.cgi?quicksearch=assignee%3Agasolin%40mozilla.com) (ASSIGNED = current working on; NEW = backlog)
 
-## 6/27 ~ 7/1
+## 7/4 ~ 7/8
 
 Firefox
 
 - [CC] Bug 1204007 - Show blocked icons in the control center when they have been blocked
   - resize svg with inkscape
-  - r?
+  - r+ & landed
 
-- [devtool] Bug 1272774 - some favicons are missing on about:debugging / tabs
-  - WIP, move code into server/actor and use client.request
+- [CC]Bug 1284172 - the indexedDB icon style shows a strange icon on Linux
+  - r+ & landed
 
-- identify bug 1193006 css svg bug related to linux
+Devtools
+
+- Contact and Meet with devtool team, will start taking track #2 tasks https://wiki.mozilla.org/DevTools/devtools-html#Track_.233_-_De-Chrome
```

Comment:
Wesly will join today's devtool's meeting. In this iteration I saw linclark and honza also involved in this track